backends-velox/src/main/java/org/apache/gluten/monitor/VeloxMemoryProfiler.java

```diff
@@ -0,0 +1,29 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.apache.gluten.monitor;
+
+public class VeloxMemoryProfiler {
+
+  /** Starts the Velox memory profiler. + jemalloc: prof.active=ture */
+  public static native void start();
+
+  /** Dumps the current memory profile. + jemalloc: prof.dump */
+  public static native void dump();
+
+  /** Stops the Velox memory profiler. + jemalloc: prof.active=false */
+  public static native void stop();
```

Comment:
Seems start & stop are not called. Am I missing something?

Reply:
> Seems start & stop are not called. Am I missing something?

Currently, we do not use them. We enable prof at executor startup via `spark.executorEnv.MALLOC_CONF prof:true,prof_prefix:/tmp/gluten_heap_perf`. We can provide them first and use them for stage-level prof later.